pages/common/ar.md

```diff
@@ -3,22 +3,22 @@
 > Create, modify, and extract from archives (`.a`, `.so`, `.o`).
 > More information: <https://manned.org/ar>.
 
-- Extract all members from an archive:
+- E[x]tract all members from an archive:
 
 `ar -x {{path/to/file.a}}`
 
-- List the members of an archive:
+- Lis[t] contents in a specific archive:
 
 `ar -t {{path/to/file.a}}`
 
-- Replace or add files to an archive:
+- [r]eplace or add specific files to an archive:
 
-`ar -r {{path/to/file.a}} {{path/to/file1.o}} {{path/to/file2.o}}`
+`ar -r {{path/to/file.a}} {{path/to/file1.ext path/to/file2.ext ...}}`
```

Comment:
I'm not sure if either of us understands what `ar` does (I sure don't lol), but aren't all object files `.o`?

Reply:
I guess that not all files are `.o` according to [this](https://www.howtoforge.com/linux-ar-command/) tutorial.

Reply:
I think [Wikipedia](https://en.wikipedia.org/wiki/Ar_(Unix)) actually explains this well:

> Today, ar is generally used only to create and update [static library](https://en.wikipedia.org/wiki/Static_library) files that the link editor or [linker](https://en.wikipedia.org/wiki/Linker_(computing)) uses and for generating .deb packages for the [Debian](https://en.wikipedia.org/wiki/Debian) family; it can be used to create archives for any purpose, but has been largely replaced by [tar](https://en.wikipedia.org/wiki/Tar_(computing)) for purposes other than static libraries.

I thought I knew about `deb`, so I was at first really confused by an archiver being used to create static libraries, I just thought they were ELF/Mach-O/..., just like dynamic libraries. But indeed, I learnt something:

```sh
$ file /opt/X11/lib/libGLU.a
/opt/X11/lib/libGLU.a: current ar archive
```

I'm still really put off by your use of `.ext`. If it can be any file, just don't include an extension. In general, I think the page needs to be rethought, beyond what you've already done, to show the clueless reader (me a few minutes ago) what this tool can be and is used for.

Reply:
I have no actual experience with `ar`. 😄 I have no ideas how to enhance this page. I can't ensure whether suggestions are correct or not. I guess we need somebody who knows this command and used it.